finesse/hardware/plugins/time/__init__.py

```diff
@@ -1 +1,9 @@
 """This module contains interfaces for querying time sources."""
+
+from finesse.hardware.manage_devices import get_device_instance
+from finesse.hardware.plugins.time.time_base import TimeBase
+
+
+def get_time_instance() -> TimeBase | None:
+    """Get the instance of the current time source if connected or None."""
+    return get_device_instance(TimeBase)
```

Comment:
Justo to confirm I get this right, this call will return the appropriate time device based on the configuration (the host time or the NTP server) regardless of not having any indication here of what device in particular should be provided, right?

Reply:
Right. It's the (arbitrary) time device set in manage devices, or the 'hardware set' yaml files. If a time device is not connected (with 'connect' button or device manager), then other parts if the code default to epoch zero time. Not sure of a more sensible default.